src/controls/listItemAttachments/ListItemAttachments.tsx

```diff
@@ -29,14 +29,13 @@ interface IPreviewImageCollection {
 
 export class ListItemAttachments extends React.Component<IListItemAttachmentsProps, IListItemAttachmentsState> {
   private _spservice: SPservice;
-  private previewImages: IPreviewImageCollection;
+  private previewImages: IPreviewImageCollection = {};
```

Comment:
@mgwojciech looks like all the changes on this file were already previously merged. Could it be from an earlier PR and you forgot to sync your branch since then?
Not an issue as I have checked the code, but something to keep in mind for future PRs to make reviews a little easier.

Reply:
Sure, I thought I pulled everything before pr, I'll pay more attention to this next time

Reply:
No problem at all, was only some files and was simple to compare.